virtio-devices/src/vhost_user/blk.rs

```diff
@@ -261,10 +380,20 @@ impl VirtioDevice for Blk {
                 pause_evt,
                 vu_interrupt_list: interrupt_list_sub,
                 slave_req_handler: None,
+                sockfd: Some(self.vhost_user_blk.as_mut().unwrap().as_raw_fd()),
+                disconnected: Some(self.disconnected.clone()),
+                activate_evt: Some(
+                    self.activate_evt
+                        .try_clone()
+                        .map_err(ActivateError::CloneEventfd)
```

Comment:
Don't you want to use `?` to return an error?

Reply:
I think you are right, I changed `.unwrap()` to `?`.